src/components/search/data/hooks.js

```diff
@@ -43,6 +43,7 @@ export const useSearchCatalogs = ({
   return searchCatalogs;
 };
 
+// const prodEnterpriseId = 'e783bb19-277f-479e-9c41-8b0ed31b4060';
```

Comment:
Could be extracted out into a `constant.js` file if its intending to be used in the future with a comment about it.